examples/django_demo/generic_foreignkey/factories.py

```diff
@@ -1,6 +1,7 @@
 from django.contrib.auth.models import Group, User
 from django.contrib.contenttypes.models import ContentType
 
+import factory
```

Comment:
Why is this needed?

Reply:
Several lines in this file use `factory.X` but the `factory` import itself was missing... eg:

https://github.com/FactoryBoy/factory_boy/blob/6b9beb9a4198b7bd01c61f3fc8a85859f2d361e7/examples/django_demo/generic_foreignkey/factories.py#L24

Reply:
Oh I see, yes good catch!

Reply:
Oh interesting, I either never realized that or long ago forgot that!